src/engine/runtime.js

```diff
@@ -105,11 +105,10 @@ class Runtime extends EventEmitter {
         this._cloneCounter = 0;
 
         /**
-         * Flag to emit a targets update at the end of a step. When target data
-         * changes, this flag is set to true.
-         * @type {boolean}
+         * List of all monitors.
          */
-        this._refreshTargets = false;
```

Comment:
Looks like a bad merge here — this removes a part of https://github.com/LLK/scratch-vm/pull/563. Please leave this in (along with the docstring).